src/components/ui/ListItem.svelte

```diff
@@ -1,10 +1,16 @@
 <script lang="ts">
   import { createEventDispatcher } from 'svelte';
+  import type { Action } from 'svelte/action';
 
   export { className as class };
   export { styleName as style };
   export let draggable: boolean = false;
 
+  export let tooltip: Action<HTMLElement, any> | undefined = undefined;
+  export let tooltipContent: any = undefined;
+  export let tooltipPlacement: string = 'top';
+  export let tooltipEnabled: boolean = false;
```

Comment:
I don't think you necessarily need to have an explicit `tooltipEnabled` flag here. You can just see if the `tooltipContent` is defined and pass that in as the boolean for the `enabled` flag.